docker-compose.yml

```diff
@@ -4,13 +4,26 @@ services:
     image: mariadb:11.4.2
     volumes:
       - db_data:/var/lib/mysql
+    ports:
+      - "3306:3306"
     restart: always
     environment:
       MYSQL_DATABASE: wordpress
       MYSQL_USER: wordpress
       MYSQL_PASSWORD: password
       MYSQL_ROOT_PASSWORD: password
 
+  phpmyadmin:
+    image: phpmyadmin/phpmyadmin:5.2.1
```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** So I tested it locally and while phpmyadmin worked well and was available on the port 8080, I noticed a platform mismatch warning in console when pulling the Docker image.

It was strange because phpmyadmin on Docker Hub is available for many platforms, including mine, `linux/arm64/v8`:


<img width="1096" alt="Screenshot 2024-07-31 at 17 41 59" src="https://github.com/user-attachments/assets/706e0849-0801-430f-8fad-06251d89292f">

It turned out that when I changed the `image` value as follows, everything started to works seamlessly:

```suggestion
    image: phpmyadmin:5.2.1
```

I'm sorry that I provided an incorrect suggestion in the first place 😅

Reply:
Oooh good catch! I even checked dockerhub and it didn't register! I've committed it and am going to merge.